### PR TITLE
Interns Model

### PR DESCRIPTION
**Pull request by Zeeson**
Feature: worked on Interns Models

### CODE REVIEW

File: models/Intern.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    required: true
+  },
+  internId:{
+    type: Number,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true
```
Comment (tonymj76) [inline models/Intern.js]: import the unique modules check admin models

File: models/Intern.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  phoneNumber: {
+    type: Number,
+    required: true
+  },
+  cvLink: {
```
Comment (tonymj76) [inline models/Intern.js]: remove the required here

File: models/Intern.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    type: String,
+    required: true,
+    default: 'blank'
```
Comment (tonymj76) [inline models/Intern.js]: default should be empty string

File: models/Intern.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  }, 
+  employmentStatus: {
+    type: String,
+    enum: ['employed', 'unemployed', 'student'],
+    default: 'unemployed'
+  },
+  about: {
+    type: String
+  },
+  registrationDate:{
```
Comment (tonymj76) [inline models/Intern.js]: why this field

**Review by tonymj76 — CHANGES_REQUESTED** [review]

File: models/Intern.js (hunk 4 above)
Comment (Zeeson) [reply models/Intern.js]: according to HNGi7 format
Screenshot 2020-07-24 at 10 33 33 AM

**Pull request closed without merging**